Engine/Graphics/DecalBuilder.cpp

```diff
@@ -264,60 +263,54 @@ char DecalBuilder::_49B790_build_decal_geometry(
 
     decal->uColorMultiplier = uColorMultiplier;
     decal->uNumVertices = 4;
-    decal->field_C14 = LightLevel;
+    decal->DimmingLevel = LightLevel;
```

Comment:
now that's a bit confusing - so it's dimming level instead of light? might be worth renaming